openhtf/core/test_record.py

```diff
@@ -48,6 +48,12 @@ class Attachment(collections.namedtuple('Attachment', 'data mimetype')):
   def sha1(self):
     return hashlib.sha1(self.data).hexdigest()
 
+  def _asdict(self):
+    # Don't include the attachment data when converting to dict.
+    return {
```

Comment:
Not all our users are on github or internal, so we'd have to communicate removing such a feature pretty loudly. But you fixed it so no communication needed!